### ISSUE TITLE
TWE: Revise Research: Research Plan Template

### ISSUE THREAD

**Issue by pandanista** [hackforla/internship#533]
### Overview
We need to update the Research Plan issue template so it has the same formatting as the most recent templates we have made.

### Action Items
We need to create a template issue for a Research Plan, so that we can create a Research Plan Template, when we are ready and maintain consistency with the org (across projects) and the guides project.

### Action Items
- [x] Copy the example from Resource 1.01
  - [x] Paste into a comment below using details to make it collapsible
  - [x] Use the following text for the summary
```
## Current draft of Research Plan Issue Template
```
- [x] Copy the example from Resource 1.01
  - [x] Use the following text for the summary
```
## Revised Research Plan Issue Template from the Guides Team
```
  - [x] Add bullet headers and divide work into sections
     - example: 
        - If the current template has
        > - [x] Go to the Google Drive, Research, 1. Research by Type, Folder
        >    - [x] Create a new folder in the drive called [number. name of type]
        - The revised template should have this text and formatting
        > - Create new Google Drive Folder
        >    - [x] Go to the Google Drive, Research, 1. Research by Type, Folder (Resource # 1.04)
        >    - [x] Create a new folder in the drive called [number. name of type] (e.g., 1. Mentor, 2. Intern) 
        >    - [x] update the link in the resources below (Resource # 2.02)
  - [x] Separate the resources into two sections
     - ```
        #### Resources for creating this issue
        ```
     - ```
        #### Resources gathered during the completion of this issue
        ```
  - [x] Make sure each resource is in order of how it appears in the instructions
  - [x] Number the resources
     - [x] Add the numbers to the action items
- [x] re-review the whole thing, to make sure 
   - [x] all the instructions make sense
   - [x] all the links resolve
   - [x] the work is listed in the most logical order
- [x] review with UX lead and get sign-off
- [x] Add the following note to a comment on the bottom of this issue
   ````
   This issue is complete the resources have been organized.

   this template will need to have another issue to revise it, so that it has a Customize Issue section that will guide the person through the replacing process. the resources will need to be reviewed after that happens.

   In that new issue there should be a checkbox at the end
   ```
   - [ ] askUX lead or product to update issue template Resource 1.03
   ```
   This issue can be closed when Research makes a new issue to do the above.
   ````
- [x] Add the `Ready for Research Lead` label
- [x] Put the issue in the Questions column
 

### Resources/Instructions
1.01 ~[TWE Research: Research Plan Issue Template, new issue](https://github.com/hackforla/internship/issues/new?assignees=&labels=feature%3A+board+%2F+google+drive+%2F+tools%2C+feature%3A+missing%2C+feature%3A+research+plan%2C+milestone%3A+missing%2C+role%3A+UI%2FUX+research%2C+size%3A+1pt&template=research--research-plan.md&title=TWE%3A+Create+Research+Plan+for+RP%5B%23%23%23%5D)~ replaced by 1.04
1.02 [Issue templates in the code](https://github.com/hackforla/internship/tree/main/.github/ISSUE_TEMPLATE)
1.03 [TWE Research: Research Plan Issue Template, file](https://github.com/hackforla/internship/blob/main/.github/ISSUE_TEMPLATE/research--research-plan.md)
1.04 [Issue: Research: Creating Research Plan Template](https://github.com/hackforla/internship/issues/new?assignees=&labels=complexity%3A+medium%2C+feature%3A+research+plan%2C+milestone%3A+missing%2C+Participant+Type%3A+missing%2C+Research+000%3A+missing%2C+role%3A+UI%2FUX+research%2C+size%3A+2pt&projects=&template=research--creating-research-plan.md&title=TWE%3A+IS%5BReplace+YY%5D%3A+RP%5BReplace+000%5D%3A+Research+Plan%3A+Create)
1.05 [New template file for Creating Research plan](https://github.com/hackforla/internship/blob/main/.github/ISSUE_TEMPLATE/research--creating-research-plan.md)

**Comment by pandanista** [comment]
@jinyan0425 Would you please update the `create research plan` template?

- I have a template for updating any existing research plan in the old format with the current format, and I believe we can make some edits on the `update research plan` template to help update this `create research plan` template.
- The key difference between `update research plan` and `create research plan` templates is: The `update research plan` template is to update the format but the research plan was already created; while the `create research plan` template is to create a research plan from scratch;
- Please see the `Update Research Plan` template in [this comment](https://github.com/hackforla/internship/issues/447#issuecomment-1624747209);
- The testing for Update Research Plan template was done in #398 in case if you need more context.

Let me know if you have any questions. Thank you!

**Comment by jinyan0425** [comment]
## Current draft of Research Plan Issue Template

<details><summary> Current draft of Research Plan Issue Template  </summary>
<p>

### Dependencies
- [ ] [Replace this with Roadmap issue #]
- [ ] A research setup structure (wiki, GitHub labels, Google Drive, etc.) has been created and approved.

### Overview
We need to create a Research Plan for [NAME OF RESEARCH PLAN] so that [RESEARCH GOAL(S)].

### Action Items
- [ ] Update this issue to make sure it's properly categorized and easy to manage
     - [ ] Under "Projects," add to the Project Management Board (helps with Project Management overview)
     - [ ] Add the milestone: [Research Plan Creation](https://github.com/hackforla/internship/milestone/29) (helps with prioritization)
     - [ ] Add label of the correct research plan number associated with this research plan (e.g., Research: RP001) (helps you find related issues in the same workflow)
     - [ ] Edit title of issue with the correct research plan number (same as prior)
     - [ ] Edit placeholders in Overview of this issue with the relevant information in square brackets. (personalizes the issue to make it easier to read). Any place you see [replace...] replace both the text and the brackets with what it is asking for (including in the front matter above but not where it says name, but not where it says title.
- [ ] Update this issue with the relevant Resources:
     - [ ] Go to the Research Output Overview Page (link in Resources)
       - [ ] find the relevant overview type wiki page. 
       - [ ] change `[Wiki [TYPE] Research Overview Page]` in the Resources below with the correct wiki page of the research overview by type.
     - [ ] Go to the Research Output Overview Page again
        - [ ] find the relevant wiki page for this specific research plan. 
        - [ ] change the `[RP__ Wiki Page]` in the Resources below with this RP's specific wiki page.
     - [ ] Find and link the folder you are going to put your research plan in  (e.g., Research by Type>2. Intern>RP6 - Intern Intake Interviews)
        - [ ] Go to the Google Drive's Research by Type Folder (link in Resources)
        - [ ] Go into the relevant Type Folder (e.g., 1. Mentor, 2. Intern, etc.) 
        - [ ] once inside, right-click on the relevant destination folder and copy the link. (e.g., RP6 - Intern Intake Interviews)
         - [ ] Add the Google Drive link in the Resources below with the link you just copied and revise the display text to include the Research Plan number (e.g., `Google Drive RP__ Folder` to `Google Drive RP008 Folder`)
- [ ] Create the Research Plan Document.
     - [ ] Make a new document using the TWE Research Plan template (link in Resources). Follow the instructions in the How To Write a Research Plan guide (link in Resources) and mark your progress in this issue.
     - [ ] Move your document from "My Drive" to the shared drive internship folder of this specific RP.
     - [ ] Name your document to be called "TWE: RP___ : [TITLE OF RESEARCH] Research Plan".
     - [ ] Update the document with the relevant information in accordance with the research roadmap document (found on the Research Overview Page).
     - [ ] Update the Table of contents (page 2) when you are finished with the document.
- [ ] Meet and sign-off research plan document with UX Research lead (by adding it to the research team meeting agenda).
- [ ] Update RP Wiki Page with any changes or corrections.
- [ ] Once research lead has signed off, add it to the PM/Research lead agenda
- [ ] sign-off by PM 
- [ ] Does this issue have any dependency checkboxes in the action items here?
   - [ ] if yes, remove dependencies that are listed below (issues that are waiting on this issue to be completed.  Sometimes they have already been made and sometimes they yet to be made).
       - [ ] close this issue
   - [ ] if no, check to see if the follow-up issues have already been made (see related structure issue).  If new issues have not been made, then apply label `ready for research lead` and define what needs to be done in a comment below
      - [ ] close the issue
      - [ ] move to `Question/Review` Column


### Resources
- [Wiki Research Output Overview Page](https://github.com/hackforla/internship/wiki/Research-Output-Overview)
- [Wiki [TYPE] Research Overview Page]
- [RP__ Wiki Page]
- [Google Drive Research by Type Folder](https://drive.google.com/drive/folders/1f5Qgq-ikT_UwcgRBuoBamqY0Wacsg9f5?usp=sharing)
- [Google Drive RP__ Folder]
- [TWE: Research Plan Template](https://docs.google.com/document/d/1SDW6n086PnSrrVSmPgQZWhlkz5_yC70vDvp7Lx7u0Wo/template/preview)
- [TWE: How to Write a Research Plan](https://docs.google.com/document/d/1Cwc0w4ZPUI8989w3jU8BW2LzLK_Tl5gHdI0VxN5ej0o/edit)

**Comment by jinyan0425** [comment]
## Revised Research Plan Issue Template from the Guides Team

<details><summary> Revised Research Plan Issue Template from the Guides Team </summary>
<p>

### Dependencies
- [ ] [Replace this with Roadmap issue #]
- [ ] A research setup structure (wiki, GitHub labels, Google Drive, etc.) has been created and approved.

### Overview
We need to create a Research Plan for [NAME OF RESEARCH PLAN] so that [RESEARCH GOAL(S)].

### Action Items
- [ ] Update this issue to make sure it's properly categorized and easy to manage
     - [ ] Under "Projects," add to the Project Management Board (helps with Project Management overview)
     - [ ] Add the milestone: [Research Plan Creation](https://github.com/hackforla/internship/milestone/29) (helps with prioritization)
     - [ ] Add label of the correct research plan number associated with this research plan (e.g., Research: RP001) (helps you find related issues in the same workflow)
     - [ ] Edit title of issue with the correct research plan number (same as prior)
     - [ ] Edit placeholders in Overview of this issue with the relevant information in square brackets. (personalizes the issue to make it easier to read). Any place you see [replace...] replace both the text and the brackets with what it is asking for (including in the front matter above but not where it says name, but not where it says title.
- [ ] Update this issue with the relevant Resources:
     - [ ] Go to the Research Output Overview Page (link in Resources)
       - [ ] find the relevant overview type wiki page. 
       - [ ] change `[Wiki [TYPE] Research Overview Page]` in the Resources below with the correct wiki page of the research overview by type.
     - [ ] Go to the Research Output Overview Page again
        - [ ] find the relevant wiki page for this specific research plan. 
        - [ ] change the `[RP__ Wiki Page]` in the Resources below with this RP's specific wiki page.
     - [ ] Find and link the folder you are going to put your research plan in  (e.g., Research by Type>2. Intern>RP6 - Intern Intake Interviews)
        - [ ] Go to the Google Drive's Research by Type Folder (link in Resources)
        - [ ] Go into the relevant Type Folder (e.g., 1. Mentor, 2. Intern, etc.) 
        - [ ] once inside, right-click on the relevant destination folder and copy the link. (e.g., RP6 - Intern Intake Interviews)
         - [ ] Add the Google Drive link in the Resources below with the link you just copied and revise the display text to include the Research Plan number (e.g., `Google Drive RP__ Folder` to `Google Drive RP008 Folder`)
- [ ] Create the Research Plan Document.
     - [ ] Make a new document using the TWE Research Plan template (link in Resources). Follow the instructions in the How To Write a Research Plan guide (link in Resources) and mark your progress in this issue.
     - [ ] Move your document from "My Drive" to the shared drive internship folder of this specific RP.
     - [ ] Name your document to be called "TWE: RP___ : [TITLE OF RESEARCH] Research Plan".
     - [ ] Update the document with the relevant information in accordance with the research roadmap document (found on the Research Overview Page).
     - [ ] Update the Table of contents (page 2) when you are finished with the document.
- [ ] Meet and sign-off research plan document with UX Research lead (by adding it to the research team meeting agenda).
- [ ] Update RP Wiki Page with any changes or corrections.
- [ ] Once research lead has signed off, add it to the PM/Research lead agenda
- [ ] sign-off by PM 
- [ ] Does this issue have any dependency checkboxes in the action items here?
   - [ ] if yes, remove dependencies that are listed below (issues that are waiting on this issue to be completed.  Sometimes they have already been made and sometimes they yet to be made).
       - [ ] close this issue
   - [ ] if no, check to see if the follow-up issues have already been made (see related structure issue).  If new issues have not been made, then apply label `ready for research lead` and define what needs to be done in a comment below
      - [ ] close the issue
      - [ ] move to `Question/Review` Column


### Resources
- [Wiki Research Output Overview Page](https://github.com/hackforla/internship/wiki/Research-Output-Overview)
- [Wiki [TYPE] Research Overview Page]
- [RP__ Wiki Page]
- [Google Drive Research by Type Folder](https://drive.google.com/drive/folders/1f5Qgq-ikT_UwcgRBuoBamqY0Wacsg9f5?usp=sharing)
- [Google Drive RP__ Folder]
- [TWE: Research Plan Template](https://docs.google.com/document/d/1SDW6n086PnSrrVSmPgQZWhlkz5_yC70vDvp7Lx7u0Wo/template/preview)
- [TWE: How to Write a Research Plan](https://docs.google.com/document/d/1Cwc0w4ZPUI8989w3jU8BW2LzLK_Tl5gHdI0VxN5ej0o/edit)

**Comment by jinyan0425** [comment]
## Draft of template issue for Creating Research Plan Template

**name:** Research: Creating Research Plan
**about:** We are creating a template for creating research plans so we can make sure all research plans are up-to-date and formatted consistently.
**title:** TWE: IS[Replace YY]: RP[Replace 000]: Research Plan: Create
**labels:** 'feature: research plan', 'feature: research', 'Participant Type: missing', 'Research 000: missing', 'size: 1pt', 'role: UI/UX research', 'milestone: missing'
**assignees:**

---
#### Instructions for creating this issue
The person who creates this issue should use links in the section "Resources for creating this issue" (i.e., 1.01, 1.02, etc.) to customize this issue.
- Customize the issue
    - [ ] Open the "Research Overview" wiki page (Resource 1.01)
    - [ ] Choose the appropriate research roadmap or research plan folder (ie., RP###) on the wiki page 
    - Use the information on the wiki page and in research roadmaps or research plans to customize this issue
    - [ ] Copy and paste entire template text in this issue into text editor (ie., "TextEdit" on _Mac_, "Wordpad" on _PC_)
    - Use Find/Replace (**Mac:** ⌘+F or **PC**: Ctrl+F) to search for and replace the keyword strings (including text and the brackets) below with the appropriate values 
        - Year 
            - [ ] Find/Replace `[Replace YY]` with research year (Two digit format, ex: "22" for year "2022") 
        - Research Plan Number
            - [ ] Find/Replace `[Replace 000]` with the research plan number (Three digit format, ex: "001" for research plan "RP001") 
        - Type of Participant
            - [ ] Find/Replace `[Replace TYPE OF PARTICIPANT]` with the participant type (ex: "Intern" or "Mentor")  
        - Research Plan Name
            - [ ] Find/Replace `[Replace TITLE OF RESEARCH]` with the research plan name on the wiki page or the research plan folder (ex: “Intern Intake Interview” or “Mentor Exit Interview”) 
    - [ ] Copy and paste the edited template from text editor back into this Github Issue, replacing the original text
    - [ ] Re-check all the checkboxes above after copying and pasting the edited template
    - [ ] Close the text editor
    - Update the issue title
        - [ ] Copy 
        ```
        TWE: IS[Replace YY]: RP[Replace 000]: Research Plan: Create
        ```
        - [ ] Choose "Edit" in the title section
        - [ ] Paste what you copied into the issue title
        - [ ] Choose "Save"
- Customize the issue settings (located in right sidebar)
    - [ ] Under Assignees, assign a Researcher if it has been determined. Otherwise, leave it blank.
    - [ ] Under Labels,  choose gear  ![Screenshot 2023-01-11 at 10 50 11 AM](https://user-images.githubusercontent.com/57029070/211904252-46521281-b8f8-495d-b31f-d06e425f0096.png), and apply the label `Research: RP[Replace 000]`
    - [ ] Under Labels, choose gear  ![Screenshot 2023-01-11 at 10 50 11 AM](https://user-images.githubusercontent.com/57029070/211904252-46521281-b8f8-495d-b31f-d06e425f0096.png), and apply the label `Participant Type: [Replace TYPE OF PARTICIPANT]`
    - [ ] Remove the `Research 000: missing` and `Participant Type: missing` labels
    - [ ] Under Projects, choose gear ![Screenshot 2023-01-11 at 10 50 11 AM](https://user-images.githubusercontent.com/57029070/211904252-46521281-b8f8-495d-b31f-d06e425f0096.png), choose `Repository`, and choose `Program Management`
    - [ ] Add the milestone: choose gear ![Screenshot 2023-01-11 at 10 50 11 AM](https://user-images.githubusercontent.com/57029070/211904252-46521281-b8f8-495d-b31f-d06e425f0096.png), and choose `Research Plan Creation`
    - [ ] Remove the label `milestone: missing`
- Remove this section
    - [ ] After all the checkboxes above are complete and the issue is ready to be worked on, delete all text below the "Instructions for Creating this Issue" (top heading) and above "Overview" (heading below) -- i.e., the issue assignee will never see this part 

**_IN PROGRESS_**

**Comment by jinyan0425** [comment]
@pandanista Ying, I created the draft based on the instructions and the format of ```Update Research Plan```.

Two minor questions:

1. Shall I add ```This issue is complete the resources have been organized. ... This issue can be closed when Research makes a new issue to do the above``` at the end of the draft or in a new comment (the first unchecked action item in the first comment)?
2. Shall we also update the `Current Status`section in the RP wiki after creating the research plan with information like ```The research plan has been created. No data have been collected``` or leave it as it is? For example, in [RP007 wiki](https://github.com/hackforla/internship/wiki/Research-Plan-7:-Onboarding-TWE-Team-volunteers), the `Current Status` section shows `This research plan was not completed in time. No data was collected.` while in in [RP025 wiki](https://github.com/hackforla/internship/wiki/Research-Plan-25:-Negative-Research-Experiences-Survey), the `Current Status` section show `work in progress`

Also please let me know if there is any issue with this draft. Thank you!

**Comment by pandanista** [comment]
Thank you for working on this issue, @jinyan0425.

To answer the two questions above:
1. Sure. Please leave a new comment to indicate that. 
2. Yes. We should include update the wiki page in the action items. 
    - We need to include both an update like `The research plan has been created. No data have been collected.` and also add the link to the research plan on the wiki page (see the Research Plan section under Assets)
        - Make sure we specify internal or external research plan in that section
        - for interns/minors, we need to have external and internal facing research plans, but we only provide external facing research plans on the wiki; 
        - for participants over 18, we only have internal facing research plans.

**Comment by jinyan0425** [comment]
> Thank you for working on this issue, @jinyan0425.
> 
> To answer the two questions above:
> 
> 1. Sure. Please leave a new comment to indicate that.
> 2. Yes. We should include update the wiki page in the action items.
>    
>    * We need to include both an update like `The research plan has been created. No data have been collected.` and also add the link to the research plan on the wiki page (see the Research Plan section under Assets)
>      
>      * Make sure we specify internal or external research plan in that section
>      * for interns/minors, we need to have external and internal facing research plans, but we only provide external facing research plans on the wiki;
>      * for participants over 18, we only have internal facing research plans.

@pandanista 

1. I will leave the comment. 
2. I added the current status updates action in the draft and updated the action items about updating the research plan link to specify the type of the plan. 
    * Is the template supposed to provide instructions on how to create an internal versus external research plan within the research plan Google Doc? This seems like a separate issue to me. 
    * In addition, if it is an external-facing research plan, should it have the same title as the internal-facing research plan, or should it be titled differently?

**Comment by jinyan0425** [comment]
This issue is complete the resources have been organized.

This template will need to have another issue to revise it, so that it has a Customize Issue section that will guide the person through the replacing process. the resources will need to be reviewed after that happens.

In that new issue there should be a checkbox at the end
```
- [ ] ask UX lead or product to update issue template Resource 1.03
```
This issue can be closed when Research makes a new issue to do the above.

**Comment by pandanista** [comment]
- Is the template supposed to provide instructions on how to create an internal versus external research plan within the research plan Google Doc? This seems like a separate issue to me.
    - Yes, it would be a separate issue that we don't need to consider the how to guide here.

**Comment by pandanista** [comment]
- In addition, if it is an external-facing research plan, should it have the same title as the internal-facing research plan, or should it be titled differently?
    - It has a slightly different title. See the research plan section on [RP006 wiki page](https://github.com/hackforla/internship/wiki/Research-Plan-6:-Intern-Intake-Interviews) for an example, and the document template is [here](https://docs.google.com/document/d/1Jw7ChKtUJIXp-2c8gqpWlyP0k1Q_Biq3Z9aUfEyZd_I/edit?usp=sharing).
        - In this template, we just need to remind assignees that if the participant type involves minors, we need to also include an external facing document. Then ask the assignee to go through the RPxxx folder to locate the external facing research plan and include it on the wiki page under the research plan section. If there is no external facing research plan when the participant type involves minors, then check in with the research team to create one, and then include it on the wiki page.
    -  Earlier I mentioned that "for interns/minors, we need to have external and internal facing research plans, but we only provide external facing research plans on the wiki", but I took another look at RP006's wiki, and both the internal and external documents are included (see screenshots below)
        - Let's check in with Product to make sure both need to be listed on wiki or just the external plan needs to be listed.
<img width="500" alt="Screenshot 2024-06-06 at 11 43 43 AM" src="https://github.com/hackforla/internship/assets/57029070/731a244e-c72f-4a52-88b7-017c3e0e6b05">

**Comment by jinyan0425** [comment]
> * In addition, if it is an external-facing research plan, should it have the same title as the internal-facing research plan, or should it be titled differently?
>   
>   * It has a slightly different title. See the research plan section on [RP006 wiki page](https://github.com/hackforla/internship/wiki/Research-Plan-6:-Intern-Intake-Interviews) for an example, and the document template is [here](https://docs.google.com/document/d/1Jw7ChKtUJIXp-2c8gqpWlyP0k1Q_Biq3Z9aUfEyZd_I/edit?usp=sharing).
>     
>     * In this template, we just need to remind assignees that if the participant type involves minors, we need to also include an external facing document. Then ask the assignee to go through the RPxxx folder to locate the external facing research plan and include it on the wiki page under the research plan section. If there is no external facing research plan when the participant type involves minors, then check in with the research team to create one, and then include it on the wiki page.
>   * Earlier I mentioned that "for interns/minors, we need to have external and internal facing research plans, but we only provide external facing research plans on the wiki", but I took another look at RP006's wiki, and both the internal and external documents are included (see screenshots below)
>     
>     * Let's check in with Product to make sure both need to be listed on wiki or just the external plan needs to be listed.
> 
> <img alt="Screenshot 2024-06-06 at 11 43 43 AM" width="500" src="https://private-user-images.githubusercontent.com/57029070/337400108-731a244e-c72f-4a52-88b7-017c3e0e6b05.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTc3MDA2MTMsIm5iZiI6MTcxNzcwMDMxMywicGF0aCI6Ii81NzAyOTA3MC8zMzc0MDAxMDgtNzMxYTI0NGUtYzcyZi00YTUyLTg4YjctMDE3YzNlMGU2YjA1LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA2MDYlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNjA2VDE4NTgzM1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWIyMjVjZjkwYjk2ZGY5ZTU4NDIwY2Y0ZmU3NmMzZWJiNzNkYmRlOTdiY2RmY2IxY2NjMzEyOGRiZDQ3OWMwOGEmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.YYkGMG9avqcPzo7gBsfRk6exa2ScH2sTBK12WX27aWM">

Hi Ying, Thanks! @pandanista 

1. I see. I will revise the template later. 
2. To clarify, does this template aim only to create an Internal Research Plan? If there is no external research plan when participants are minors, this template only asks the assignee to check with the research team to create one and includes the link (vs. to create one by himself)? That means, the basic actions are (they are not actual items):

- If the research involves minors:
  - [ ] If there is an external research plan, add the link
  - [ ] If there is no external research plan, ask the research team to create one
  - [ ] Once the external research plan is created, add the link

4. Such external research plans may not be located in the RP000's research folder. For example, the [external research plan for RP006](https://docs.google.com/document/d/11owu6zVKn2KkonqngAef5wXYQi3OEe6Q6x6fqn8Huts/edit?usp=drive_link) locates in a different folder:
<img width="1143" alt="Screenshot 2024-06-06 at 12 10 22 PM" src="https://github.com/hackforla/internship/assets/90875339/0769a66e-3a95-4902-807d-1f1dc1766602">
5. Yes, I also noticed that some RPs included both versions.

**Comment by pandanista** [comment]
- To clarify, does this template aim only to create an Internal Research Plan? If there is no external research plan when participants are minors, this template only asks the assignee to check with the research team to create one and includes the link (vs. to create one by himself)?
    - Yes, and yes.
- Such external research plans may not be located in the RP000's research folder. For example, the [external research plan for RP006](https://docs.google.com/document/d/11owu6zVKn2KkonqngAef5wXYQi3OEe6Q6x6fqn8Huts/edit?usp=drive_link) locates in a different folder
    - Good catch, in that case, we need to direct the assignee to locate the external facing research plan via the path shown in your screenshot. 
- Yes, I also noticed that some RPs included both versions.
    - Do you mind listing those RP's wiki page in a comment, so we could confirm with Product at the next all team meeting? Thank you, @jinyan0425.

**Comment by jinyan0425** [comment]
@pandanista 

These are RPs with both external and internal versions on RP wiki:

1. [RP006:-Intern-Intake-Interviews](https://github.com/hackforla/internship/wiki/Research-Plan-6:-Intern-Intake-Interviews)
2. [RP012:-Intern-Exit-Interviews](https://github.com/hackforla/internship/wiki/Research-Plan-12:-Intern-Exit-Interviews)
3. [RP013:-Mentor-Intake-Survey](https://github.com/hackforla/internship/wiki/Research-Plan-13:-Mentor-Intake-Survey)
4. [RP016:-Intern-Productivity-Dashboard-Survey](https://github.com/hackforla/internship/wiki/Research-Plan-16:-Intern-Productivity-Dashboard-Survey)

**Comment by jinyan0425** [comment]
> * To clarify, does this template aim only to create an Internal Research Plan? If there is no external research plan when participants are minors, this template only asks the assignee to check with the research team to create one and includes the link (vs. to create one by himself)?
>   
>   * Yes, and yes.
> * Such external research plans may not be located in the RP000's research folder. For example, the [external research plan for RP006](https://docs.google.com/document/d/11owu6zVKn2KkonqngAef5wXYQi3OEe6Q6x6fqn8Huts/edit?usp=drive_link) locates in a different folder
>   
>   * Good catch, in that case, we need to direct the assignee to locate the external facing research plan via the path shown in your screenshot.
> * Yes, I also noticed that some RPs included both versions.
>   
>   * Do you mind listing those RP's wiki page in a comment, so we could confirm with Product at the next all team meeting? Thank you, @jinyan0425.

@pandanista 
1. I looked at all RPs and found no external research plan for IS21 (e.g., RP001-005). As for IS22, all external research plan are located in [IS22 Cohort - External Communication - External Communication - Parent folder](https://drive.google.com/drive/folders/1uKw7lUS-lNiL5cYSUlZPi-98VmcnNiT-?usp=drive_link). This may create some confusion if this template only guides the assignee to check this folder, esp. when we have future cohorts. 
   - Instead, maybe we can consider moving all external research plans to the corresponding RP folder first? It might be confusing that internal research plans are in folders organized by RP numbers while external research plans are in folders organized by cohort year.
   - I will update the template once everything is clear.
3.  Comment made, see above.

**Comment by pandanista** [comment]
> This issue is complete the resources have been organized.
> 
> This template will need to have another issue to revise it, so that it has a Customize Issue section that will guide the person through the replacing process. the resources will need to be reviewed after that happens.
> 
> In that new issue there should be a checkbox at the end
> 
> ```
> - [ ] ask UX lead or product to update issue template Resource 1.03
> ```
> 
> This issue can be closed when Research makes a new issue to do the above.

@jinyan0425 I took another look at the template, and I believe the customize the issue setting part is already included. So I am gonna hide that comment.

**Comment by pandanista** [comment]
- We would like to clarify on the following questions:
    - Under what circumstances do we need to create an external facing research plan?
    - Do we always include both internal and external research plans on the research plan's wiki?
    - Why is external research plan not saved in the corresponding research folder? Asking this because we want to make sure we are setting up the most appropriate path for external and internal research plans storage going forward.
        - For example, for IS22, all external research plan are located in [IS22 Cohort - External Communication - External Communication - Parent folder](https://drive.google.com/drive/folders/1uKw7lUS-lNiL5cYSUlZPi-98VmcnNiT-?usp=drive_link)

Thank you.

**Comment by ExperimentsInHonesty** [comment]
@pandanista See my answers below each of your questions

> Under what circumstances do we need to create an external facing research plan?

A: We always include an external research plan for all research plans that involve user interview and surveys

> Do we always include both internal and external research plans on the research plan's wiki?

A: YES


>Why is external research plan not saved in the corresponding research folder? Asking this because we want to make sure we are setting up the most appropriate path for external and internal research plans storage going forward.  Example, for IS22, all external research plan are located in [IS22 Cohort - External Communication - External Communication - Parent folder](https://drive.google.com/drive/folders/1uKw7lUS-lNiL5cYSUlZPi-98VmcnNiT-?usp=drive_link)

A: Because at the time the research plan folders were private (you have to be a member to access) and we don't provide access on a document by document basis, so we put all public facing documents in a folder that is open to the public and that way, any document added is automatically public

**Comment by jinyan0425** [comment]
@pandanista Ying, I updated the template based on our discussion. I removed the action items about the external-facing RP and finished the part about updating the WIKI page. Please let me know if you have any questions or suggestions.

**Comment by pandanista** [comment]
It looks good to me. We will review with product and will let you know if we have any questions. Thank you, @jinran.

**Comment by ExperimentsInHonesty** [comment]
The template has been updated in the repo
